rust-sdk/core/src/math/tick_array.rs

```diff
@@ -298,8 +296,7 @@ mod tests {
     #[test]
     fn test_get_prev_initialized_tick_out_of_bounds() {
         let sequence = test_sequence(16);
-        let pair = sequence.prev_initialized_tick(-1409);
-        assert_eq!(pair.map(|x| x.1), Ok(-1408));
-        assert_eq!(pair.map(|x| x.0), Ok(None));
```

Comment:
```
test sequence: [-1408      0) [0         1408) [1408          2816)
               start = -1408                                    end=2815
```
Then
- `sequence.prev_initialized_tick(-1407) -> Ok((None, -1408))`
- `sequence.prev_initialized_tick(-1408) -> Ok((None, -1408))` # prev is inclusive
- `sequence.prev_initialized_tick(-1409) -> Err(INVALID_TICK_ARRAY_SEQUENCE )`
- `sequence.prev_initialized_tick(-1410) -> Err(INVALID_TICK_ARRAY_SEQUENCE )`